Commandant/Errors.swift

```diff
@@ -39,7 +39,11 @@ internal func informativeUsageError<T>(keyValueExample: String, option: Option<T
 		description += "]"
 	}
 
-	description += "\n\t\(option.usage)"
+	description += option.usage.componentsSeparatedByString("\n")
+		.reduce(""){ previous, value in
+			return previous + "\n\t" + value
+		}
```

Comment:
No, join will only insert "\n\t" between the elements, the reduce will prepend every element with it (while concatenating).